core/device_hooks/common_kernels.inc.cpp

```diff
@@ -198,6 +199,38 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_TYPE(GKO_DECLARE_BICGSTAB_STEP_3_KERNEL);
 }  // namespace bicgstab
 
 
+namespace cgs {
```

Comment:
the user has to do this.

Reply:
@tcojean do you think it's possible to do this automatically with a script

Since this file is rarely changed, my idea would be to generate a "patch template", with placeholders for the solver name, some dummy entries, just to show what format to use, and a coment line `// TODO(SolverGenerator): instructions` or something similar that the user can `grep` to see where the things have to be changed.

Then we can use this template to generate a patch, and apply it using the `patch` command in the script.